src/model/entities/test/entity-test-factory.ts

```diff
@@ -76,6 +76,33 @@ export class EntityTestFactory {
     }
   }
 
+  public static createINewsGatewayDevice(device: Partial<INewsGatewayDevice> = {}): INewsGatewayDevice {
+    return {
+      id: 'deviceId',
+      name: 'deviceName',
+      statusCode: StatusCode.UNKNOWN,
+      statusMessage: '',
+      isConnected: false,
+      type: DeviceType.INEWS_GATEWAY,
+      username: 'testuser',
+      password: 'testpassword',
```

Comment:
For the sake of brevity and completing this sprint I will ignore this for now.

Reply:
Aaaaand it turns out I cannot ignore this, since the queues params list is needed to connect.
Okay. To paraphrase King Henry during the siege of Harfleur:

"Once more unto the breach, dear friends, once more;
Or close the wall up with our developer dead."